addons/sys_sem70/functions/fnc_onDisplayButtonPress.sqf

```diff
@@ -0,0 +1,30 @@
+/*
+ * Author: ACRE2Team
+ * SHORT DESCRIPTION
+ *
+ * Arguments:
+ * 0: ARGUMENT ONE <TYPE>
+ * 1: ARGUMENT TWO <TYPE>
+ *
+ * Return Value:
+ * RETURN VALUE <TYPE>
+ *
+ * Example:
+ * [ARGUMENTS] call acre_COMPONENT_fnc_FUNCTIONNAME
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params["_control"];
```

Comment:
I'll stop commenting about those here, should just take a search and replace thing anyways to convert all of those.